content/altinn-studio/guides/development/signing/role-based-signing/backend-manual/add-process-task.en.md

```
@@ -56,14 +67,19 @@ This data type is used to store the actual signatures that are generated when a
 It is important to set `allowedContributors` to `"app:owned"`. This ensures that these data items cannot be edited via the app’s API but only by the app itself. Before version 8.6, this was misspelled `allowedContributers`.
 
-The ID can be set to something else, but they must match the IDs entered in `signatureDataType` in the process step, as shown in the process task step.
+We recommend placing [access restrictions](/altinn-studio/guides/development/restricted-data/) on the signature objects, by specifying the `actionRequiredToRead` and `actionRequiredToWrite` properties on the data type definition. If this is not something you wish to do, those properties can be removed from the configuration.
 
-### Tilgangsstyring
+#### Access control
+Give `read`, `write` and alternatively `sign` to the instance owner. Others who should sign also need `read` and `write`.
 
-Give `read`, `write` and alternatively `sign` to the instance owner. Others who should sign also need `read` and `write`.
\ No newline at end of file
+More information about action attributes can be found [here](/altinn-studio/reference/configuration/authorization/#action-attributes).
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)